### ISSUE TITLE
[Link] Component specs

### ISSUE THREAD

**Issue by aweell** [dxc-technology/halstack-style-guide#221]
Documentation:

- [x] Add focus state 
- [x] Remove unstyled link
- [x] To improve accessibility all states should have underline

**Comment by aweell** [comment]
Closed via #255
